app/views/questionnaire.py

```diff
@@ -258,7 +263,7 @@ def get_thank_you(eq_id, form_type):  # pylint: disable=unused-argument
 def get_view_submission(eq_id, form_type):  # pylint: disable=unused-argument
 
     session_data = get_session_store().session_data
-    g.schema = load_schema_from_session_data(session_data)
+    g.schema = schema = load_schema_from_session_data(session_data)
```

Comment:
Pretty cool, had no idea that you could do chain assignments in python!

Reply:
Really useful. Don't make the mistake of assigning the same reference to multiple objects and thinking they're different objects.

Pretty obvious:
```
>>> a_string = b_string = "foo"
>>> a_string = "bar"
>>> a_string
'bar'
>>> b_string
'foo'
```

Sometimes less expected:
```
>>> a_list = b_list = []
>>> a_list.append('foo')
>>> a_list
['foo']
>>> b_list
['foo']
```